worker/src/main/java/gov/cms/ab2d/worker/config/JobHandler.java

```diff
@@ -40,6 +43,8 @@ public void handleMessage(Message<?> message) throws MessagingException {
 
         final String jobId = getJobId(message);
 
+        MDC.put(JOB_LOG, jobId);
```

Comment:
This `MDC` will _not_ get cleared by the Filter I discussed earlier; so we need to clear it manually. So add `MDC.remove()` to the end of this method.